src/main/java/org/folio/oaipmh/processors/MarcWithHoldingsRequestHelper.java

```diff
@@ -680,6 +677,7 @@ private ResumptionTokenType buildResumptionTokenFromRequest(Request request, Str
     extraParams.put(OFFSET_PARAM, String.valueOf(cursor + returnedCount));
     extraParams.put(REQUEST_ID_PARAM, requestId);
     extraParams.put(NEXT_RECORD_ID_PARAM, nextInstanceId);
+    extraParams.put(EXPIRATION_DATE_RESUMPTION_TOKEN, String.valueOf(Instant.now().plusSeconds(60)));
```

Comment:
60 seconds looks magic number. Save it as constant.

Reply:
done
https://github.com/folio-org/mod-oai-pmh/pull/223/commits/2938681ecdcd8ccf194cbc2da3479c7a27eea0b8